docs/getting-started-guides/docker-multinode/skydns.yaml.in

```diff
@@ -1,31 +1,35 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: kube-dns-v8
+  name: kube-dns-v10
   namespace: kube-system
   labels:
     k8s-app: kube-dns
-    version: v8
+    version: v10
     kubernetes.io/cluster-service: "true"
 spec:
   replicas: {{ pillar['dns_replicas'] }}
   selector:
     k8s-app: kube-dns
-    version: v8
+    version: v10
   template:
     metadata:
       labels:
         k8s-app: kube-dns
-        version: v8
+        version: v10
         kubernetes.io/cluster-service: "true"
     spec:
       containers:
       - name: etcd
-        image: gcr.io/google_containers/etcd:2.0.9
+        image: gcr.io/google_containers/etcd-amd64:2.2.1
         resources:
+          # keep request = limit to keep this container in guaranteed class
           limits:
             cpu: 100m
             memory: 50Mi
+          requests:
```

Comment:
I've just copied this from https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/skydns-rc.yaml.in to keep the files up-to-date
@gmarek changed it in #18216